<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>391</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_control">
    <attribute name="title">
     <string>Control</string>
    </attribute>
    <widget class="QGroupBox" name="group_setting">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>341</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Channels</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Frequency</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button_setup">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>310</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SETUP</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="button_checkAll">
         <property name="text">
          <string>CHECK ALL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_ch1">
         <property name="text">
          <string>CH 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_ch2">
         <property name="text">
          <string>CH 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_ch3">
         <property name="text">
          <string>CH 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_ch4">
         <property name="text">
          <string>CH 4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_ch5">
         <property name="text">
          <string>CH 5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_ch6">
         <property name="text">
          <string>CH 6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_ch7">
         <property name="text">
          <string>CH 7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_ch8">
         <property name="text">
          <string>CH 8</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>83</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="chkbox_freq4">
         <property name="text">
          <string>4000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_freq8">
         <property name="text">
          <string>8000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_freq16">
         <property name="text">
          <string>16000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkbox_freq32">
         <property name="text">
          <string>32000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>151</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>interval (minutes)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edit_interval"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Deadline (days)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edit_deadline"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Experiment Name</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edit_name"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_checkchip">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>200</width>
       <height>327</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>CheckChip</string>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>181</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_checkchip">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>CHECKCHIP</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_device">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>221</width>
       <height>341</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Device</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>State :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>End Date :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Elapsed Time :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button_pause">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PAUSE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_start">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_report">
    <attribute name="title">
     <string>Report</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
